src/eq_schema/Answer.test.js

```diff
@@ -135,6 +135,51 @@ describe("Answer", () => {
       ]);
     });
 
+    it("should should always put Exclusive options at the end", () => {
+      const answer = new Answer(
+        createAnswerJSON({
+          type: "Checkbox",
+          options: [
+            {
+              id: 1,
+              label: "Option one",
+              description: "A short description"
+            },
+            {
+              id: 2,
+              label: "Option two",
+              description: "Another description"
+            }
+          ],
+          mutuallyExclusiveOption: {
+            id: 3,
+            label: "Option three",
+            description: "I am mutually Exclusive"
+          }
+        })
+      );
+
+      expect(answer.type).toEqual("MutuallyExclusiveCheckbox");
+
+      expect(answer.options).toEqual([
+        {
+          label: "Option one",
+          value: "Option one",
+          description: "A short description"
+        },
+        {
+          label: "Option two",
+          value: "Option two",
+          description: "Another description"
+        },
+        {
+          label: "Option three",
+          value: "Option three",
+          description: "I am mutually Exclusive"
+        }
+      ]);
+    });
+
```

Comment:
Probably worth adding a test with mutually exclusive option and an `other` option to ensure that other appears before the mutually exclusive option.